src/qrl/core/OutgoingMessage.py

```diff
@@ -6,6 +6,16 @@
 
 
 class OutgoingMessage:
+    """
+    The meaning of Message here is different from
+    Message/MessageReceipt/MessageRequest. Here, the Message is anything that
+    P2PProtocol sends. (Yes, something needs to be renamed)
```

Comment:
```suggestion
    P2PProtocol sends. (Yes, something needs to be renamed)
```
No it doesnt needs renaming.